services/src/pro/users/postgres_userdb.rs

```diff
@@ -432,4 +432,43 @@ where
 
         Ok(())
     }
+
+    async fn increment_quota_used(&self, user: &UserId, quota_used: u64) -> Result<()> {
+        let conn = self.conn_pool.get().await?;
+        let stmt = conn
+            .prepare("UPDATE users SET quota_used = quota_used + $1 WHERE id = $2;")
+            .await?;
```

Comment:
Can we keep the prepared message? It is somehow strange that we prepare it X times. Or is there a cache from the DB side?

Reply:
I don't understand the problem. What would the alternative here, string concatenation?

> Prepared statements can only be used with the connection that created them.

We are using a connection pool so if we wanted to cache them we would need one cache per connection. I don't think this is a) feasible, b) a problem and c) part of this user story.